github/create-github-release/create-github-release.yaml

```diff
@@ -0,0 +1,62 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: create-github-release
+spec:
+  workspaces:
+    - name: input
+    - name: source
+    - name: release-notes
+  params:
+    - name: TAG
+      description: A git tag that will be created with this release (e.g. v1.0.0)
+      type: string
+    - name: REVISION
+      type: string
+      description: Git revision to create a release from (branch, tag, sha, ref…).
+      default: master
+    - name: RELEASE_FILE_NAME
+      type: string
+      description: Name of the file that has to be uploaded as release notes.
+      default: release.md
+    - name: GITHUB_TOKEN_SECRET
+      type: string
+      description: Name of the secret holding the github-token.
+      default: github-token
+    - name: GITHUB_TOKEN_SECRET_KEY
+      type: string
+      description: Name of the secret key holding the github-token.
+      default: GITHUB_TOKEN
+  steps:
+    - name: create-release
+      workingdir: $(workspaces.source.path)
+      image: gcr.io/tekton-releases/dogfooding/hub
```

Comment:
OK, apologies one further wrinkle: we probably shouldn't rely on one of Tekton's infra images since we don't want to be responsible for modifications there breaking a catalog task.  Looks like there are several publicly available images for `hub`:

- https://hub.docker.com/r/kanisterio/alpine-git-hub
- https://hub.docker.com/r/tianon/github-hub/

Or it might be a good time to build a fresh image and publish it to docker hub / quay to support the specific needs of this Task?

Reply:
Yeah sure, I have build the image and updated the task. Also added link to the `Dockerfile` in the `README.md`